<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeskGUI</class>
 <widget class="QMainWindow" name="DeskGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Путь коня.</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Координаты конечной позиции</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_pos">
      <property name="text">
       <string>b5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="desk">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_knight">
      <property name="text">
       <string>Координаты коня</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit_knight">
      <property name="text">
       <string>a3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
